### ISSUE TITLE
test_ip_address_ipv6_ipv4_mapped is failing

### ISSUE THREAD

**Issue by sudipm-mukherjee** [kellyjonbrazil/jc#604]
In Debian the test for `test_ip_address_ipv6`_ipv4_mapped is now failing.
I see this in the logs:
```
======================================================================
FAIL: test_ip_address_ipv6_ipv4_mapped (tests.test_ip_address.MyTests.test_ip_address_ipv6_ipv4_mapped)
Test ipv6 address with ipv4 mapped string
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/<<PKGBUILDDIR>>/.pybuild/cpython3_3.12_jc/build/tests/test_ip_address.py", line 94, in test_ip_address_ipv6_ipv4_mapped
    self.assertEqual(jc.parsers.ip_address.parse(data, quiet=True), expected)
AssertionError: {'ver[787 chars]ed': False, 'is_unspecified': False, 'int': {'[1595 chars]11'}} != {'ver[787 chars]ed': True, 'is_unspecified': False, 'int': {'i[1594 chars]11'}}
Diff is 2775 characters long. Set self.maxDiff to None to see it.

----------------------------------------------------------------------
```

**Comment by kellyjonbrazil** [comment]
I added some fixes for similar issues that have cropped up in Python 3.12 and 3.13 and backports on the `dev` branch. This might be a new one I have to look into.

**Comment by kellyjonbrazil** [comment]
I have fixed this in the `dev` branch:

https://github.com/kellyjonbrazil/jc/blob/dev/tests/test_ip_address.py#L87

**Comment by sudipm-mukherjee** [comment]
I think I am missing something.  I am assuming you meant this commit at https://github.com/kellyjonbrazil/jc/commit/ba6e1e694e7fd3865945dad1e462eb94a2bba478 but that is not fixing the error in Debian.

**Comment by sudipm-mukherjee** [comment]
This is with the latest  test_ip_address.py from `dev` branch.

```
======================================================================
FAIL: test_ip_address_ipv6_ipv4_mapped (tests.test_ip_address.MyTests.test_ip_address_ipv6_ipv4_mapped)
Test ipv6 address with ipv4 mapped string
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/<<PKGBUILDDIR>>/.pybuild/cpython3_3.13_jc/build/tests/test_ip_address.py", line 146, in test_ip_address_ipv6_ipv4_mapped
    self.fail('\n\n'.join(failure_msgs))
    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: New-style IPv4 Mapped is_reserved=false failure:
  Expected:  {'version': 6, 'max_prefix_length': 128, 'ip': '::ffff:192.168.1.35', 'ip_compressed': '::ffff:192.168.1.35', 'ip_exploded': '0000:0000:0000:0000:0000:ffff:192.168.1.35', 'ip_split': ['0000', '0000', '0000', '0000', '0000', 'ffff', '192', '168', '1', '35'], 'scope_id': None, 'ipv4_mapped': '192.168.1.35', 'six_to_four': None, 'teredo_client': None, 'teredo_server': None, 'dns_ptr': '3.2.1.0.8.a.0.c.f.f.f.f.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa', 'network': '::ffff:192.168.1.35', 'broadcast': '::ffff:192.168.1.35', 'hostmask': '::', 'netmask': 'ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff', 'cidr_netmask': 128, 'hosts': 1, 'first_host': '::ffff:192.168.1.35', 'last_host': '::ffff:192.168.1.35', 'is_multicast': False, 'is_private': True, 'is_global': False, 'is_link_local': False, 'is_loopback': False, 'is_reserved': False, 'is_unspecified': False, 'int': {'ip': 281473913979171, 'network': 281473913979171, 'broadcast': 281473913979171, 'first_host': 281473913979171, 'last_host': 281473913979171}, 'hex': {'ip': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'network': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'broadcast': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'hostmask': '00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00', 'netmask': 'ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff', 'first_host': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'last_host': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23'}, 'bin': {'ip': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'network': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'broadcast': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'hostmask': '00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000', 'netmask': '11111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111', 'first_host': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'last_host': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011'}}
  Actual:  {'version': 6, 'max_prefix_length': 128, 'ip': '::ffff:192.168.1.35', 'ip_compressed': '::ffff:192.168.1.35', 'ip_exploded': '0000:0000:0000:0000:0000:ffff:192.168.1.35', 'ip_split': ['0000', '0000', '0000', '0000', '0000', 'ffff', '192.168.1.35'], 'scope_id': None, 'ipv4_mapped': '192.168.1.35', 'six_to_four': None, 'teredo_client': None, 'teredo_server': None, 'dns_ptr': '3.2.1.0.8.a.0.c.f.f.f.f.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa', 'network': '::ffff:192.168.1.35', 'broadcast': '::ffff:192.168.1.35', 'hostmask': '::', 'netmask': 'ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff', 'cidr_netmask': 128, 'hosts': 1, 'first_host': '::ffff:192.168.1.35', 'last_host': '::ffff:192.168.1.35', 'is_multicast': False, 'is_private': True, 'is_global': False, 'is_link_local': False, 'is_loopback': False, 'is_reserved': True, 'is_unspecified': False, 'int': {'ip': 281473913979171, 'network': 281473913979171, 'broadcast': 281473913979171, 'first_host': 281473913979171, 'last_host': 281473913979171}, 'hex': {'ip': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'network': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'broadcast': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'hostmask': '00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00', 'netmask': 'ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff', 'first_host': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'last_host': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23'}, 'bin': {'ip': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'network': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'broadcast': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'hostmask': '00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000', 'netmask': '11111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111', 'first_host': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'last_host': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011'}}

New-style IPv4 Mapped is_reserved=true failure:
  Expected:  {'version': 6, 'max_prefix_length': 128, 'ip': '::ffff:192.168.1.35', 'ip_compressed': '::ffff:192.168.1.35', 'ip_exploded': '0000:0000:0000:0000:0000:ffff:192.168.1.35', 'ip_split': ['0000', '0000', '0000', '0000', '0000', 'ffff', '192', '168', '1', '35'], 'scope_id': None, 'ipv4_mapped': '192.168.1.35', 'six_to_four': None, 'teredo_client': None, 'teredo_server': None, 'dns_ptr': '3.2.1.0.8.a.0.c.f.f.f.f.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa', 'network': '::ffff:192.168.1.35', 'broadcast': '::ffff:192.168.1.35', 'hostmask': '::', 'netmask': 'ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff', 'cidr_netmask': 128, 'hosts': 1, 'first_host': '::ffff:192.168.1.35', 'last_host': '::ffff:192.168.1.35', 'is_multicast': False, 'is_private': True, 'is_global': False, 'is_link_local': False, 'is_loopback': False, 'is_reserved': True, 'is_unspecified': False, 'int': {'ip': 281473913979171, 'network': 281473913979171, 'broadcast': 281473913979171, 'first_host': 281473913979171, 'last_host': 281473913979171}, 'hex': {'ip': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'network': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'broadcast': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'hostmask': '00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00', 'netmask': 'ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff', 'first_host': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'last_host': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23'}, 'bin': {'ip': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'network': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'broadcast': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'hostmask': '00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000', 'netmask': '11111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111', 'first_host': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'last_host': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011'}}
  Actual:  {'version': 6, 'max_prefix_length': 128, 'ip': '::ffff:192.168.1.35', 'ip_compressed': '::ffff:192.168.1.35', 'ip_exploded': '0000:0000:0000:0000:0000:ffff:192.168.1.35', 'ip_split': ['0000', '0000', '0000', '0000', '0000', 'ffff', '192.168.1.35'], 'scope_id': None, 'ipv4_mapped': '192.168.1.35', 'six_to_four': None, 'teredo_client': None, 'teredo_server': None, 'dns_ptr': '3.2.1.0.8.a.0.c.f.f.f.f.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa', 'network': '::ffff:192.168.1.35', 'broadcast': '::ffff:192.168.1.35', 'hostmask': '::', 'netmask': 'ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff', 'cidr_netmask': 128, 'hosts': 1, 'first_host': '::ffff:192.168.1.35', 'last_host': '::ffff:192.168.1.35', 'is_multicast': False, 'is_private': True, 'is_global': False, 'is_link_local': False, 'is_loopback': False, 'is_reserved': True, 'is_unspecified': False, 'int': {'ip': 281473913979171, 'network': 281473913979171, 'broadcast': 281473913979171, 'first_host': 281473913979171, 'last_host': 281473913979171}, 'hex': {'ip': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'network': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'broadcast': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'hostmask': '00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00', 'netmask': 'ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff', 'first_host': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'last_host': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23'}, 'bin': {'ip': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'network': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'broadcast': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'hostmask': '00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000', 'netmask': '11111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111', 'first_host': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'last_host': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011'}}

Old-style IPv4 Mapped is_reserved=false failure:
  Expected:  {'version': 6, 'max_prefix_length': 128, 'ip': '::ffff:c0a8:123', 'ip_compressed': '::ffff:c0a8:123', 'ip_exploded': '0000:0000:0000:0000:0000:ffff:c0a8:0123', 'ip_split': ['0000', '0000', '0000', '0000', '0000', 'ffff', 'c0a8', '0123'], 'scope_id': None, 'ipv4_mapped': '192.168.1.35', 'six_to_four': None, 'teredo_client': None, 'teredo_server': None, 'dns_ptr': '3.2.1.0.8.a.0.c.f.f.f.f.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa', 'network': '::ffff:c0a8:123', 'broadcast': '::ffff:c0a8:123', 'hostmask': '::', 'netmask': 'ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff', 'cidr_netmask': 128, 'hosts': 1, 'first_host': '::ffff:c0a8:123', 'last_host': '::ffff:c0a8:123', 'is_multicast': False, 'is_private': True, 'is_global': False, 'is_link_local': False, 'is_loopback': False, 'is_reserved': False, 'is_unspecified': False, 'int': {'ip': 281473913979171, 'network': 281473913979171, 'broadcast': 281473913979171, 'first_host': 281473913979171, 'last_host': 281473913979171}, 'hex': {'ip': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'network': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'broadcast': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'hostmask': '00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00', 'netmask': 'ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff', 'first_host': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'last_host': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23'}, 'bin': {'ip': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'network': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'broadcast': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'hostmask': '00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000', 'netmask': '11111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111', 'first_host': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'last_host': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011'}}
  Actual:  {'version': 6, 'max_prefix_length': 128, 'ip': '::ffff:192.168.1.35', 'ip_compressed': '::ffff:192.168.1.35', 'ip_exploded': '0000:0000:0000:0000:0000:ffff:192.168.1.35', 'ip_split': ['0000', '0000', '0000', '0000', '0000', 'ffff', '192.168.1.35'], 'scope_id': None, 'ipv4_mapped': '192.168.1.35', 'six_to_four': None, 'teredo_client': None, 'teredo_server': None, 'dns_ptr': '3.2.1.0.8.a.0.c.f.f.f.f.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa', 'network': '::ffff:192.168.1.35', 'broadcast': '::ffff:192.168.1.35', 'hostmask': '::', 'netmask': 'ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff', 'cidr_netmask': 128, 'hosts': 1, 'first_host': '::ffff:192.168.1.35', 'last_host': '::ffff:192.168.1.35', 'is_multicast': False, 'is_private': True, 'is_global': False, 'is_link_local': False, 'is_loopback': False, 'is_reserved': True, 'is_unspecified': False, 'int': {'ip': 281473913979171, 'network': 281473913979171, 'broadcast': 281473913979171, 'first_host': 281473913979171, 'last_host': 281473913979171}, 'hex': {'ip': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'network': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'broadcast': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'hostmask': '00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00', 'netmask': 'ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff', 'first_host': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'last_host': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23'}, 'bin': {'ip': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'network': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'broadcast': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'hostmask': '00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000', 'netmask': '11111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111', 'first_host': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'last_host': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011'}}

Old-style IPv4 Mapped is_reserved=true failure:
  Expected:  {'version': 6, 'max_prefix_length': 128, 'ip': '::ffff:c0a8:123', 'ip_compressed': '::ffff:c0a8:123', 'ip_exploded': '0000:0000:0000:0000:0000:ffff:c0a8:0123', 'ip_split': ['0000', '0000', '0000', '0000', '0000', 'ffff', 'c0a8', '0123'], 'scope_id': None, 'ipv4_mapped': '192.168.1.35', 'six_to_four': None, 'teredo_client': None, 'teredo_server': None, 'dns_ptr': '3.2.1.0.8.a.0.c.f.f.f.f.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa', 'network': '::ffff:c0a8:123', 'broadcast': '::ffff:c0a8:123', 'hostmask': '::', 'netmask': 'ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff', 'cidr_netmask': 128, 'hosts': 1, 'first_host': '::ffff:c0a8:123', 'last_host': '::ffff:c0a8:123', 'is_multicast': False, 'is_private': True, 'is_global': False, 'is_link_local': False, 'is_loopback': False, 'is_reserved': True, 'is_unspecified': False, 'int': {'ip': 281473913979171, 'network': 281473913979171, 'broadcast': 281473913979171, 'first_host': 281473913979171, 'last_host': 281473913979171}, 'hex': {'ip': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'network': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'broadcast': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'hostmask': '00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00', 'netmask': 'ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff', 'first_host': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'last_host': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23'}, 'bin': {'ip': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'network': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'broadcast': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'hostmask': '00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000', 'netmask': '11111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111', 'first_host': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'last_host': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011'}}
  Actual:  {'version': 6, 'max_prefix_length': 128, 'ip': '::ffff:192.168.1.35', 'ip_compressed': '::ffff:192.168.1.35', 'ip_exploded': '0000:0000:0000:0000:0000:ffff:192.168.1.35', 'ip_split': ['0000', '0000', '0000', '0000', '0000', 'ffff', '192.168.1.35'], 'scope_id': None, 'ipv4_mapped': '192.168.1.35', 'six_to_four': None, 'teredo_client': None, 'teredo_server': None, 'dns_ptr': '3.2.1.0.8.a.0.c.f.f.f.f.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa', 'network': '::ffff:192.168.1.35', 'broadcast': '::ffff:192.168.1.35', 'hostmask': '::', 'netmask': 'ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff', 'cidr_netmask': 128, 'hosts': 1, 'first_host': '::ffff:192.168.1.35', 'last_host': '::ffff:192.168.1.35', 'is_multicast': False, 'is_private': True, 'is_global': False, 'is_link_local': False, 'is_loopback': False, 'is_reserved': True, 'is_unspecified': False, 'int': {'ip': 281473913979171, 'network': 281473913979171, 'broadcast': 281473913979171, 'first_host': 281473913979171, 'last_host': 281473913979171}, 'hex': {'ip': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'network': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'broadcast': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'hostmask': '00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00', 'netmask': 'ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff:ff', 'first_host': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23', 'last_host': '00:00:00:00:00:00:00:00:00:00:ff:ff:c0:a8:01:23'}, 'bin': {'ip': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'network': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'broadcast': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'hostmask': '00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000', 'netmask': '11111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111', 'first_host': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011', 'last_host': '00000000000000000000000000000000000000000000000000000000000000000000000000000000111111111111111111000000101010000000000100100011'}}

----------------------------------------------------------------------

```

**Comment by kellyjonbrazil** [comment]
Ah, I see - there were also changes I made to `ip_split` in the parser due to the Python changes.

https://github.com/kellyjonbrazil/jc/commit/71af0c55553e17c598c9f43b8271bfb069795af5#diff-a225634674c8982b6195f7f9309ace86102c88af78fbf75ca4f8c5f40a73c786L665-R676

If the older `ip_address` parser is being used I'll probably have to fix the tests to account for that.

**Comment by kellyjonbrazil** [comment]
Now available in release v1.25.4
